### PR TITLE
CASSANDRA-19148: Remove unused dead code

### PR DESCRIPTION
**Pull request by yifan-c**

### CODE REVIEW

File: cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/utils/CqlUtils.java [hunk 1]
```diff
@@ -47,7 +46,6 @@ public final class CqlUtils
 {
     // Properties to be overridden when extracted from the table schema
     private static final List<String> TABLE_PROPERTY_OVERRIDE_ALLOWLIST = Arrays.asList("bloom_filter_fp_chance",
-                                                                                        "cdc",
```
Comment (frankgh) [inline cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/utils/CqlUtils.java:50]: Yeah, I think this is no longer needed here.

**Review by frankgh — APPROVED** [review]
+1 Looks good. CI is also green

**Comment by jberragan** [comment]
+1

**Comment by frankgh** [comment]
Closed via https://github.com/apache/cassandra-analytics/commit/680cc9395c55a88217f2de975f62ad588e8c95d5

**Pull request closed without merging**